From: David Steele <dsteele@gmail.com>
Date: Mon, 14 Apr 2014 21:21:45 -0400
Subject: Set Python path to load the right daemon

Some users are having a problem with the wrong daemon module
being loaded. Modify the Python path to ensure that
python-daemon is the one found.

This uses Debian conventions, and is therefore Debian-only.
---
 cloudprint/cloudprint.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cloudprint/cloudprint.py b/cloudprint/cloudprint.py
index 1eaebc0..46ee367 100755
--- a/cloudprint/cloudprint.py
+++ b/cloudprint/cloudprint.py
@@ -595,6 +595,10 @@ def main():
 
     if args.daemon:
         try:
+            # workaround to avoid overloaded module
+            prepath = '/usr/lib/pymodules/python%s.%s' % sys.version_info[0:2]
+            sys.path.insert(0, prepath)
+
             import daemon
             import daemon.pidfile
         except ImportError:
